scripts/update/jellyfin.sh

```diff
@@ -68,15 +62,79 @@ if [[ -f /install/.jellyfin.lock ]]; then
     fi
     #
     if ! check_installed jellyfin; then
-        echo_info "Moving Jellyfin to apt-managed installation"
+        echo_info "Updating Jellyfin installation using apt."
         #
-        # Add the jellyfin official repository and key to our installation so we can use apt-get to install it jellyfin and jellyfin-ffmepg.
-        curl -s https://repo.jellyfin.org/$DIST_ID/jellyfin_team.gpg.key | gpg --dearmor > /usr/share/keyrings/jellyfin-archive-keyring.gpg 2>> "${log}"
-        echo "deb [signed-by=/usr/share/keyrings/jellyfin-archive-keyring.gpg arch=$(dpkg --print-architecture)] https://repo.jellyfin.org/$DIST_ID $DIST_CODENAME main" > /etc/apt/sources.list.d/jellyfin.list
+        # Make sure universe is enabled so that ffmpeg can be satisfied.
+        sudo add-apt-repository universe
```

Comment:
This is ubuntu-centric code that doesn't work on Debian.

Also, all Ubuntu repos are guaranteed to be installed:

https://github.com/swizzin/swizzin/blob/master/scripts/update/10-dependencies.sh#L8-L46